test/jdk/java/awt/TrayIcon/DisposeInActionEventTest/DisposeInActionEventTest.java

```diff
@@ -81,8 +82,10 @@ private static JFrame showFrameAndIcon() {
         JFrame frame = new JFrame("DisposeInActionEventTest");
```

Comment:
```suggestion
        JFrame frame = new JFrame("Event Message Display");
```
This is where events are displayed.